bftengine/src/bftengine/ViewsManager.hpp

```diff
@@ -71,6 +71,9 @@ class ViewsManager {
                                                                           // in viewChangeMsgs
                                         std::vector<ViewChangeMsg *> viewChangeMsgs);
 
+  ViewNum getCurrentView() const { return myCurrentView; }
+  void setHigherView(ViewNum higherViewNum);
+  void setViewFromRecovery(ViewNum explicitViewNum) { myCurrentView = explicitViewNum; }
```

Comment:
`setCurrentView()` ?

Reply:
The name of this function aims to state that it is called only when the replica is recovering. In this certain scenario the view number might be set to a specific value, which is not strictly higher than the one before (i.e. it might be the same value).